src/types/onyx/DismissedProductTraining.ts

```diff
@@ -6,6 +6,26 @@ type DismissedProductTraining = {
      * When user dismisses the nudgeMigration Welcome Modal, we store the timestamp here.
      */
     migratedUserWelcomeModal: Date;
+
+    /**
+     * When user dismisses the conciergeLHNGBR product training tooltip, we store the timestamp here.
+     */
+    conciergeLHNGBR: Date;
```

Comment:
Could we use the constants from Const file for these? Or do we not typically do that for types files?

Reply:
Yes exactly. What do you think?